.circleci/config.yml

```diff
@@ -461,11 +414,15 @@ workflows:
             tags:
               <<: *filters-release-tags
           context:
-            - dockerhub-quorumengineering-ro             
+            - dockerhub-quorumengineering-ro
       - extractAPISpec:
           requires:
             - assemble
           filters:
+            branches:
+              only:
+                - master
+                - /^release-.*/
```

Comment:
This doesn't seem right.  We should be extracting the API spec on PRs, just not publishing them. Otherwise if the extract is broken we don't find out until after it merges.